projects/storefrontstyles/scss/components/product/list/_facet-list.scss

```diff
@@ -1,3 +1,52 @@
+%cx-dialog {
```

Comment:
One small thing, why is the close button on the dialog only visible in mobile/tablet?

Reply:
in inline mode, you can't close the facets. It's always there ;)